### PR TITLE
Cleans up card generation a little

### PR DESCRIPTION
**Pull request by ShizCalev**
Just condensing some code down, nothing really to note.

### CODE REVIEW

**Comment by Cyprex** [comment]
Ace can be added to the persons for loop

**Comment by francinum** [comment]
Not without breaking the order.

**Comment by Cyprex** [comment]
This seems to change the order anyway?

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -678,29 +678,15 @@
...
-	cards += "Jack of Spades"
-	cards += "Jack of Clubs"
-	cards += "Jack of Diamonds"
-	cards += "Ace of Hearts"
-	cards += "Ace of Spades"
-	cards += "Ace of Clubs"
-	cards += "Ace of Diamonds"
+	for(var/suit in list("Hearts", "Spades", "Clubs", "Diamonds"))
+		cards += "Ace of [suit]"
+		for(var/i in (2 to 10))
```
Comment (actioninja) [inline code/game/objects/items/toys.dm]: this is setting off the linter, it's a known issue
```suggestion
		for(var/i in 2 to 10)
```

**Review by actioninja — CHANGES_REQUESTED** [review]

**Comment by SpaceManiac** [comment]
7 day stale

**Pull request merged**